### PR TITLE
Add support for querying Elasticsearch aliases

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

**Comment by synhershko** [comment]
This looks good :+1: 

I'd also add a way to query index prefixes / wildcards, which is a common use-case with ES, so `SELECT Count(*) FROM "logs-*"` would work (maybe it already does now)

**Comment by martint** [comment]
>  I'd also add a way to query index prefixes / wildcards, which is a common use-case with ES, so SELECT Count(*) FROM "logs-*" would work (maybe it already does now)

That's currently not supported, but we'll do that as a separate change. I need to investigate a bit how to handle the metadata for those kinds of "virtual" tables.

File: presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java [hunk 1]
```diff
@@ -473,6 +475,49 @@ public void testQueryStringError()
...
+                .indices()
+                .aliases(indexAliasesRequest()
+                        .addAliasAction(IndicesAliasesRequest.AliasActions.add()
+                                .index("region")
+                                .alias("multi_alias")))
+                .actionGet();
+
+        assertQuery("" +
+                        "SELECT count(*) FROM multi_alias",
+                "SELECT (SELECT count(*) FROM region) + (SELECT count(*) FROM nation)");
```
Comment (ebyhr) [inline presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java]: nit: We can remove `"" +` as other assertQuery in this class 
```java
        assertQuery(
                        "SELECT count(*) FROM multi_alias",
                "SELECT (SELECT count(*) FROM region) + (SELECT count(*) FROM nation)");
```

**Review by ebyhr — APPROVED** [review]

**Comment by synhershko** [comment]
> That's currently not supported, but we'll do that as a separate change. I need to investigate a bit how to handle the metadata for those kinds of "virtual" tables.

@martint I thought about this again, and it's actually perfectly fine to require an alias to be defined if the user wants to query an index pattern (prefixes, wildcards, etc). It's just a matter of documentation now.

**Pull request merged**